xla/service/gpu/transforms/cudnn_fused_conv_rewriter.cc

```diff
@@ -640,14 +670,16 @@ void CaptureConvGraphRecursive(HloInstruction* instr,
                 m::Broadcast(&op0, m::ConstantEffectiveScalar(0)).WithOneUser(),
                 m::Op().Is(instr),
                 m::Broadcast(&operand0, m::ConstantEffectiveScalar(6))
-                    .WithOneUser()))) {
+                    .WithOneUser())) &&
+        eligible_operand(operand0)) {
```

Comment:
operand0 is constant, so it cannot possibly have a dependency on an aux output. And similarly for leaky relu.